spec/controllers/posts_controller_spec.rb

```diff
@@ -1,23 +1,23 @@
 require 'rails_helper'
 
-RSpec.describe 'Posts requests', type: :request do
-  describe 'Index method' do
+RSpec.describe 'posts_controller', type: :request do
+  describe 'index action' do
     before(:example) { get user_posts_path(user_id: 1) }
 
     it 'returns the list of posts for a given user' do
-      expect(response).to have_http_status(200)
+      expect(response).to have_http_status(:ok)
     end
 
-    it 'renders the correct template for a list of posts for a given user' do
+    it 'renders the correct template' do
       expect(response).to render_template(:index)
     end
   end
 
-  describe 'Show method' do
+  describe 'show action' do
     before(:example) { get user_post_path(user_id: 1, id: 1) }
 
     it 'returns the correct data of a post from a given user' do
-      expect(response).to have_http_status(200)
+      expect(response).to have_http_status(:ok)
     end
 
```

Comment:
- [ ] Kindly most of your tests are not passing. Kindly fix them.

![image](https://user-images.githubusercontent.com/65068771/173169330-484227db-8b77-4a3f-b408-f0595e9ed93b.png)

Reply:
It is because the test environment is empty, I just filled it and it works fine for me :grin: 

![image](https://user-images.githubusercontent.com/78335447/173190361-adb2dc61-e773-4cce-89b8-05312ecbf164.png)